### DISCUSSION TITLE
How to close current pdf?

### DISCUSSION THREAD

**Discussion by JunglePredator** [pdfarranger/pdfarranger#663] [Q&A] [answered]
If I'm processing multiple PDFs & want to close the current one to open a new one... the only way I see is to completely close pdf arranger?

**Comment by jeromerobert** [comment, marked as answer]
This is https://github.com/pdfarranger/pdfarranger/issues/593. It's already implemented and will be available in release 1.9.0. I encourage you to test that upcoming release by installing the main branch of PDF-Arranger. See https://github.com/pdfarranger/pdfarranger#install-from-source and https://github.com/pdfarranger/pdfarranger/actions/runs/2213349677 for Windows.
